src/content/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NerdGraph tutorial: Longer-running asynchronous NRQL queries"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Use New Relic NerdGraph to make asynchronous NRQL queries that have longer query duration limits.
+---
+
+If you want to run NRQL queries that have a 10-minute [query duration limit](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries), you can use [our NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to run an asynchronous query. This is useful for queries of a large data set or that query over a long time range. It can also be useful for running queries that have less risk of being interrupted by browser or HTTP connection timeout limits or other issues. 
+
+## Requirements [#requirements]
+
+Requires [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-plus). 
```

Comment:
I'll defer to you on this, but to be clear: Data Plus isn't a requirement for asynchronous queries, it's just a requirement for query durations that extend all the way out to 10 minutes. To me, this is a little confusing, because it reads like I need Data Plus to run async queries at all. But I also realize from the title that this doc is mainly concerned with 10-minute queries, so maybe it's fine as is.

Reply:
ah i see, right, sorry